### PR TITLE
Power shell examples import branch

### PR DESCRIPTION
**Pull request by SteveWilkins1123**
﻿

Fixes #

### Changes proposed in this pull request

-
-
-

### Other links

-
-
-

### CODE REVIEW

**Review by peombwa — APPROVED** [review]
~~LGTM!~~

File: src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md [hunk 1]
```diff
@@ -1,18 +1,30 @@
-### Example 1: {{ Add title here }}
+### Example 1
```
Comment (peombwa) [inline src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md:1]: Missing title:
```suggestion
### Example 1: {{ TODO: Add title here }}
```

File: src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md [hunk 2]
```diff
@@ -1,18 +1,30 @@
...
+		Usage = "Verify"
+		Key = [System.Text.Encoding]::ASCII.GetBytes("MIIDYDCCAki...")
+	}
+	PasswordCredential = $null
+	Proof = "eyJ0eXAiOiJ..."
+}
+Add-MgApplicationKey -ApplicationId $applicationId -BodyParameter $params
 ```
-
-{{ Add description here }}
```
Comment (peombwa) [inline src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md:8]: Missing description:
```
{{ TODO: Add description here }}

```

**Review by peombwa — CHANGES_REQUESTED** [review]
@SteveWilkins1123, the PR validation is failing due to:
1. Missing title for the examples.
2. Missing description for the examples.

See [Get-MgDomainFederationConfiguration.md](https://github.com/MicrosoftGraph/msgraph-sdk-powershell/blob/dev/src/Identity.DirectoryManagement/Identity.DirectoryManagement/examples/v1.0/Get-MgDomainFederationConfiguration.md) example.

File: src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md (hunk 2 above)
Comment (SteveWilkins1123) [reply src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md:8]: Are you failing because the descriptions are missing?

Comment (SteveWilkins1123) [reply src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md:8]: And as a rule they mostly have Example 1 Example 2 etc as the title

Comment (peombwa) [reply src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md:8]: The validation rule is by Azure's AutoREST. It expects all examples to have a tile and description. See https://github.com/Azure/autorest.powershell/blob/1a1c6d89c2b07be22fcada5cd3a13e92b1a7e1c7/powershell/resources/assets/tools/Resources/how-to.md#generating-documentation

**Comment by peombwa** [comment]
@msewaweru, thoughts on the generated examples in this PR? The examples are extracted from the snippets in our API reference docs.

**Comment by SteveWilkins1123** [comment]
Integration taking too long

**Comment by msewaweru** [comment]
> @msewaweru, thoughts on the generated examples in this PR? The examples are extracted from the snippets in our API reference docs.

Hi @peombwa , I will review the examples in the subsequent PRs now that this has been closed. Thanks.

**Pull request closed without merging**